components/splitbutton/stories/splitbutton.stories.js

```diff
@@ -16,9 +16,19 @@ export default {
       options: ["s", "m", "l", "xl"],
       control: "select"
     },
+    variant: {
+      name: "Variant",
+      type: { name: "string", required: true },
```

Comment:
```suggestion
      type: { name: "string" },
```

Q: Does split button render okay if no variant class exists on it? If so, let's remove the required flag.